### PR TITLE
Update Compaction Strategy example

### LINKED ISSUE #82: Compaction Strategy Example is no longer accurate
The compaction strategy example should be updated to reflect the recent changes to compaction configuration in accumulo. Including the description of the strategies and the example configuration given. 
Example below of what could be added to replace the current example (this is found I'm accumulo-testing)
<pre>
tserver.compaction.major.service.cs1.planner=org.apache.accumulo.core.spi.compaction.DefaultCompactionPlanner \
tserver.compaction.major.service.cs1.planner.opts.executors=\
[{"name":"small","type":"internal","maxSize":"16M","numThreads":8},\
{"name":"medium","type":"internal","maxSize":"128M","numThreads":4},\
{"name":"large","type":"internal","numThreads":2
</pre>

The blog post on the accumulo website also gives further details on how to configure the new compactions. 
https://accumulo.apache.org/blog/2021/07/08/external-compactions.html

### PR DESCRIPTION
**Pull request by jmark99**
Update the Compaction Strategy example to make use of the new Compaction Configuration API.

Closes #82

### CODE REVIEW

**Review by Manno15 — APPROVED** [review]
Changes look good. Confirmed the example works and the amount of detail is vastly improved. I assume this example will also be updated to the main branch as well?

File: docs/compactionStrategy.md [hunk 1]
```diff
@@ -16,64 +16,137 @@ limitations under the License.
...
  - Compact files less than 100M using snappy.
- 
-```bash
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.file.compress.type=snappy"
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.majc.compaction.strategy=org.apache.accumulo.tserver.compaction.strategies.BasicCompactionStrategy"
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.majc.compaction.strategy.opts.filter.size=250M"
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.majc.compaction.strategy.opts.large.compress.threshold=100M"
- $ accumulo shell -u <username> -p <password> -e "config -t examples.test1 -s table.majc.compaction.strategy.opts.large.compress.type=gz"
-```
+ - Limit the compaction throughput to 40M.
```
Comment (dlmarion) [inline docs/compactionStrategy.md]: Below you have this as `40MB/s`. Using that here would make this more clear.

**Review by dlmarion — COMMENTED** [review]
Looks good, one suggested change to make things clear about the rate limit.

File: docs/compactionStrategy.md (hunk 1 above)
Comment (jmark99) [reply docs/compactionStrategy.md]: @dlmarion , with this clarification updated, are you good with me merging this example?

Comment (dlmarion) [reply docs/compactionStrategy.md]: Yes I believe so. Looks good. Thanks!

**Review by dlmarion — APPROVED** [review]

**Pull request merged**